docs/docs/pods.md

```diff
@@ -24,7 +24,7 @@ Currently, Marathon pods can only be created and administered via the `/v2/pods/
 1. Run the following REST call, substituting your IP and port for `<ip>` and `<port>`:
 
     ```bash
-    $ curl -X POST -H "Content-type: application/json" -d@/dev/stdin http://<ip>:<port>/v2/pods <<EOF
+    $ curl -X POST -H "Content-type: application/json" -d@<mypod>.json http://<ip>:<port>/v2/pods <<EOF
```

Comment:
nope, this particular one was fine using `/dev/stdin` (because it's got `<<EOF` at the end).

now, it's wrong